### ISSUE TITLE
Make mobile hamburger toggle and close button tabbable: 0.5

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#268]
_Epic: #265_

> The mobile menu is currently not accessible because they hamburger toggle is not tabbable. Once opened the close button is also not tabbable.

![screen shot 2018-04-27 at 3 04 14 pm](https://user-images.githubusercontent.com/263285/39382777-4ad13576-4a2c-11e8-970f-efc7471e72d8.png)

WCAG 2.0 Compliance: ???

Traceability: [Microassist Report 1 > WCAG Compliance > Name, Role, Value](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.t8cmbkue4bcm)

**Comment by jessesnyder** [comment]
@paregorios I think there's actually more that should be done here, but the tabbing problem from this ticket is addressed.

**Comment by paregorios** [comment]
@jessesnyder I'd be grateful if you could take a few minutes and look through the other tickets linked from our Epic "make the mobile menu fully accessible" (#265) and alert me to any [concerns you've noticed](https://github.com/isawnyu/isaw.web/issues/268#issuecomment-432000440) while working on this ticket that are **not** covered by existing tickets. Thanks!

**Comment by paregorios** [comment]
Please merge this change to master and queue for deployment.

**Comment by jessesnyder** [comment]
@paregorios The other tickets in the epic do already address the issues I was concerned about (aria labels and screen reader text, primarily). We're good (or will be)!
